lib/logic.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module TicTacToe
+  class Game
+    attr_accessor :player1, :player2, :board
```

Comment:
It isn't allowed to use `attr_accessor`, you should use either `attr_reader` or `attr_writer`

Reply:
Resolved also. :)